lib/driver.js

```diff
@@ -1749,10 +1746,27 @@ class XCUITestDriver extends BaseDriver {
       /** @type {string[]} */ (appsList).map((app) => this.helpers.configureApp(app, '.app'))
     );
     for (const otherApp of appPaths) {
-      // @ts-expect-error - do not assign arbitrary properties to `this.opts`
-      await installToSimulator(this.opts.device, otherApp, undefined, {
-        newSimulator: this.lifecycleData.createSim,
-      });
+      this.isRealDevice()
+        ? await installToRealDevice(
```

Comment:
Please only use a ternary operator if it is needed to get a result of an expression. Otherwise a vanilla if condition does the job just fine.